src/client/utils/countries_transform.js

```diff
@@ -0,0 +1,12 @@
+export default function(countries)
+{
```

Comment:
js lint won't pass bracket under statement. Now we do not have it in supplier but as only we will (matter of days) it will crash